lib/modules/platform/bitbucket/index.ts

```diff
@@ -192,11 +199,14 @@ export async function initRepo({
   // TODO #7154
   const hostnameWithoutApiPrefix = regEx(/api[.|-](.+)/).exec(hostname!)?.[1];
 
+  const auth: string = opts.token
```

Comment:
```suggestion
  const auth = opts.token
```

typescript shoud be smart enough 🙃

Reply:
The `token` property doesn't exist on the [InternalHttpOptions interface](https://github.com/renovatebot/renovate/blob/main/lib/util/http/types.ts#L70). Let me know if I should take a different strategy